documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc

```diff
@@ -6,25 +6,39 @@
 [id='deploying-the-topic-operator-using-the-cluster-operator-{context}']
 = Deploying the Topic Operator using the Cluster Operator
 
+This procedure describes how to deploy the Topic Operator using the Cluster Operator. Alternatively, you can deploy the Topic Operator as a standalone component. For more information, see xref:deploying-the-topic-operator-standalone[].
```

Comment:
I think we should clarify when should the alternative be used. The standalone deployment of the Topic Operator should be used when the user wants to use it with Kafka cluster not managed by Strimzi.